joomla-projects/media-manager-improvement
# Imagecropper fixes

#### Summary of Changes

Updated the imagecropper library to the latest version
Fixed the toolbar buttons
Cleaned up the file to use Joomla conventions
Fixed Javascript functions
#### Testing Instructions

There are two ways to test any PR here:
- The easy one where you need to download a copy of this repo from [here](https://github.com/joomla-projects/media-manager-improvement/archive/master.zip) and then install it locally. Also grab com_patchtester and install that too and then go to it's configuration options and set the project name to joomla-projects and the repo to media-manager-improvement
  After that a fetch in the com_patchtester will show all the PRs and you can apply any of those and test it.
- The hard way needs some git commands where you get that repo locally and then switch to any branch. Again you will have to install it locally pointing your servers path to the folder that the local repo exists 

Basically, try to crop an image and see if it works.
#### NOTE:

Right now there is an inconsistency in the upload logic (php side) for files with a `-` which will be converted to `_`. So please check the created files carefully
#### Final remarks

Although this feature isn't fully crystallized I think it is usable as-is for testing.
